src/inmanta/module.py

```diff
@@ -611,6 +698,25 @@ def __init__(self, local_repo: "ModuleRepo", remote_repo: "ModuleRepo") -> None:
         self.local_repo: ModuleRepo = local_repo
         self.remote_repo: ModuleRepo = remote_repo
 
+    def log_pre_install_information(self, project: "Project", module_name: str) -> None:
+        LOGGER.debug("Installing module %s (v1).", module_name)
```

Comment:
I think this would be the perfect place to add [constraints information](https://github.com/inmanta/inmanta-core/pull/4554#discussion_r923126078) (same for v2). I'll get back to your question there ASAP.